### ISSUE TITLE
Possible bug using TextBox hints

### ISSUE THREAD

**Issue by mgnslndh** [ButchersBoy/MaterialDesignInXamlToolkit#251]
I have encountered a situation where the hints of a TextBox gets displayed even though the TextBox has a value in the Text property. Everything is OK until I navigate away from the view and then back. 

I am using ReactiveUI for navigation. Don't know if that is possibly related. I guess the view is discarded and recreated between the first and second time the view is displayed.

My XAML:

```
<TextBox Grid.Column="0" Grid.Row="0" Margin="8,8,8,8" FontSize="24" HorizontalContentAlignment="Left" VerticalAlignment="Bottom" mdt:TextFieldAssist.Hint="NEEDLE SENSITIVITY" behaviors:TextBoxBehavior.SelectAllOnFocus="True">
                        <TextBox.Style>
                            <Style TargetType="TextBox" BasedOn="{StaticResource MaterialDesignFloatingHintTextBox}">
                                <Setter Property="Text" Value="{Binding NeedleSensitivity, StringFormat={}{0} %, UpdateSourceTrigger=PropertyChanged}" />
                                <Style.Triggers>
                                    <Trigger Property="Control.IsFocused" Value="True">
                                        <Setter Property="Text" Value="{Binding NeedleSensitivity, StringFormat={}{0}, UpdateSourceTrigger=LostFocus}" />
                                    </Trigger>
                                </Style.Triggers>
                            </Style>
                        </TextBox.Style>
                        <i:Interaction.Behaviors>
                            <behaviors:NumericTextBoxBehavior AllowDecimal="False" AllowNegatives="False"  />
                        </i:Interaction.Behaviors>
                    </TextBox>
```

Sorry for the crude GIF animation but it shows the TextBoxes with hints displayed correctly, then navigation to another page and back. When the TextBoxes are displayed the second time the hints and Text content are dislayed at the same time.

![hintbug](https://cloud.githubusercontent.com/assets/1139118/12939990/5355d7c6-cfc5-11e5-9914-aa4b9dc98a59.gif)

**Comment by ButchersBoy** [comment]
Are you using latest?

**Comment by mgnslndh** [comment]
No, I am using 1.4.0.473. You are updating too fast for me :) Thats good. I'll check out the latest stable version.

**Comment by ButchersBoy** [comment]
Closing this as should be fixed in 1.4.1...if you find otherwise, re-open.

**Comment by leovictor** [comment]
I had the same issue but only when i load the data from database. If I clean the text and input from keyboard it didn't happen! I'm using 1.4.1.485 but i tried 1.5.0-ci496 and it happened too.
